assets/optional/operator-lifecycle-manager/release-aarch64.json

```diff
@@ -0,0 +1,10 @@
+{
+  "release": {
+    "base": ""
```

Comment:
Should this file be empty when committed into the repo?

Reply:
I think that this PR should already include outputs of `rebase.sh images` so all artifacts commited to main are functional. Is it possible yet?

Reply:
not sure. I use the image-references values to replace all the other instances of those images in any of the manifests. Isn't rebase.sh images called before generating the RPMs?